projects/libpsl/build.sh

```diff
@@ -17,6 +17,7 @@
 
 # avoid iconv() memleak on Ubuntu 16.04 image (breaks test suite)
 export ASAN_OPTIONS=detect_leaks=0
+export LD_LIBRARY_PATH=/usr/local/lib
```

Comment:
During build time /usr/local/lib/libidn2.so is linked (as reference) into the executable (proved with 'ldd' command locally).
That works out for execution as well (due to the absolute path).
I successfully tested locally with `helper.py build_fuzzers` and `helper.py run_fuzzer`.